frontend/public/components/resource-pages.ts

```diff
@@ -83,7 +84,7 @@ const addDynamicResourcePage = (
   map: ImmutableMap<ResourceMapKey, ResourceMapValue>,
   page: DynamicResourcePage,
 ) => {
-  const key = referenceForExtensionModel(page.properties.model);
+  const key = referenceForExtensionModel(page.properties.model as ExtensionK8sModel);
```

Comment:
I don't like this cast, can we fix the accepted type to be: `{ group?: ..., version?: ..., kind?: ...} ` ?